.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Typos and doc fixes
+about: Typos and doc fixes
+title: ''
+labels: documentation
+assignees: ''
+---
+
+## 📚 Documentation
+
+For typos and doc fixes, please go ahead and:
+
+1. Create an issue (You can skip this if it is a very simple fix).
+1. Fix the documentation.
+1. Make sure documentation builds without an error (See [Building the docs](https://pyodide.org/en/stable/development/contributing.html#building-the-docs)).
+1. Submit a PR.
```

Comment:
I think in most cases it's fine if people submit the PR directly and check the rendering in CI. 

Then for larger PRs mention that they can indeed build the docs locally if needed.